src/tests/suite-ui/editor-driver.test.ts

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
I also don't understand the point of this file, couldn't we just point to the documentation/samples for the `vscode-extension-tester` testing framework instead of having these tests?

Reply:
It's good to have at least 2 test files in the suite to make sure each test file cleans up properly.  This was a generic test used for reference and to make sure things got cleaned up.  I guess its purpose has been served, and I removed it.